modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaConfigurationModelParser.java

```diff
@@ -193,8 +199,6 @@ public Optional<DisplayModel> getDisplayModel() {
 
   @Override
   public Optional<MuleVersion> getMinMuleVersion() {
-    return of(calculateConfigMinMuleVersion(configElement,
-                                            getContainerAnnotationMinMuleVersion(extensionElement, Configurations.class,
-                                                                                 Configurations::value, configElement)));
+    return of(this.minMuleVersion);
```

Comment:
This trusts that the calculation will always retrieve a value, what about using ofNullable?